llvm/utils/TableGen/Basic/IntrinsicEmitter.cpp

```diff
@@ -526,6 +526,11 @@ static AttributeSet getIntrinsicArgAttributeSet(LLVMContext &C, unsigned ID) {
 )",
                     ID);
       for (const CodeGenIntrinsic::ArgAttribute &Attr : Attrs) {
+        if (Attr.Kind == CodeGenIntrinsic::NoCapture) {
```

Comment:
Should NoCapture->CapturesNone (or something like that) here as well for consistency? Although might further increase the diff :)

Reply:
I'd prefer doing that in a followup, to avoid a mass-change of Intrinsic.td files in this PR. I think we'll also want to expose at least CapturesRetOnly, as some intrinsics want those semantics. Not sure if any intrinsics need something other than CapturesNone/CapturesRetOnly -- exposing the attribute in full generality in TableGen would probably be rather annoying.

Reply:
Sounds good, as long as it will be renamed eventually.